content/_code-samples/use-tickets/py/use-tickets.py

```diff
@@ -0,0 +1,54 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import TicketCreate, AccountSet
+from xrpl.transaction import safe_sign_and_submit_transaction
+from xrpl.wallet import Wallet, generate_faucet_wallet
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Generate a wallet and request faucet
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Construct a TicketCreate transaction, 1 ticket created for future use
+tx = TicketCreate(
+    account=myAddr,
+    ticket_count=1
+)
+
+# Sign transaction locally and submit
+my_tx_payment_signed = safe_sign_and_submit_transaction(transaction=tx, wallet=test_wallet, client=client)
+
+# Get a Ticket Sequence
+get_ticket_sequence = AccountObjects(
+    account=myAddr,
+    type=AccountObjectType.TICKET
+)
+
+response = client.request(get_ticket_sequence)
+ticket_sequence = response.result["account_objects"][0]["TicketSequence"]
+
+# Construct Transaction using a Ticket
+tx_1 = AccountSet(
+    account=myAddr,
+    fee="10",
+    sequence=0,
+    last_ledger_sequence=None,
+    ticket_sequence=ticket_sequence
```

Comment:
when running the script - the error "TypeError: AccountSet.__init__() got an unexpected keyword argument 'ticket_sequence'" showed up

Reply:
@mDuo13: I had the same error as well BUT I had to modify xrpl-py's code on AccountSet.init() (added ticket_sequence field) and it worked.

Reply:
xrpl-py is missing this common field in its latest release. It will be fixed with the next one:
Issue:
https://github.com/XRPLF/xrpl-py/issues/413

PR:
https://github.com/XRPLF/xrpl-py/pull/428

So `ticket_sequence` is correct :)